configs/defs/fallback.textproto

```diff
@@ -5491,140 +5701,1902 @@ benchmark_configs: {
+      "        }"
+      "      }"
+      "    }"
+      " }"
+      " check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{path:\"/boot/grub/menu.lst\"}}"
+      "      content_entry:{"
+      "        match_type: ALL_MATCH_ANY_ORDER"
+      "        match_criteria: {"
+      "          filter_regex: \"\\\\s*kernel.*audit=[0-9]+.*\""
+      "          expected_regex: \"\\\\s*kernel.*audit=1.*\""
+      "        }"
+      "      }"
+      "    }"
+      " }"
+      "}"
+  }
+}
+benchmark_configs: {
+  id: "events-modifying-date-time-info-collected-fallback"
+  compliance_note: {
+    version: { cpe_uri: "fallback" version: "2.0.0" benchmark_document: "CIS Distribution Independent Linux" }
+    title: "Ensure events that modify date and time information are collected"
+    description:
+      "Capture events where the system date and/or time has been modified. "
+      "The parameters in this section are set to determine if the adjtimex (tune kernel clock), "
+      "settimeofday (Set time, using timeval and timezone structures) stime (using seconds since 1/1/1970) "
+      "or clock_settime (allows for the setting of several internal clocks and timers) "
+      "system calls have been executed and always write an audit record to the /var/log/audit.log "
+      "file upon exit, tagging the records with the identifier \"time-change\""
+    rationale:
+      "Unexpected changes in system date and/or time could be a sign of malicious activity on the system."
+    remediation:
+      "For 32 bit systems Edit or create a file in the /etc/audit/rules.d/ directory ending in .rules \n"
+      "Example: vi /etc/audit/rules.d/audit.rules \n"
+      "and add the following lines: \n"
+      "```-a always,exit -F arch=b32 -S adjtimex -S settimeofday -S stime -k timechange``` \n"
+      "-a always,exit -F arch=b32 -S clock_settime -k time-change \n"
+      "-w /etc/localtime -p wa -k time-change ```\n"
+      "For 64 bit systems Edit or create a file in the /etc/audit/rules.d/ directory ending in .rules \n"
+      "Example: vi /etc/audit/rules.d/audit.rules \n"
+      "and add the following lines: \n"
+      "```-a always,exit -F arch=b64 -S adjtimex -S settimeofday -k time-change \n"
+      "-a always,exit -F arch=b32 -S adjtimex -S settimeofday -S stime -k time-change \n"
+      "-a always,exit -F arch=b64 -S clock_settime -k time-change \n"
+      "-a always,exit -F arch=b32 -S clock_settime -k time-change \n"
+      "-w /etc/localtime -p wa -k time-change ```\n"
+      "Notes: \n"
+      "Reloading the auditd config to set active settings may require a system reboot."
+    cis_benchmark: {
+      profile_level: 2
+      severity: LOW
+    }
+    scan_instructions:
+      "scan_type_specific:{"
+      "    instance_scanning:{"
+      "        check_alternatives:{"
+      "            file_checks:{"
+      "                files_to_check:{single_file:{path: \"/etc/audit/audit.rules\"}}"
```

Comment:
There are a couple more checks here and below where we'd fail if it ran on a binary-only container - let's add check_alternatives for the base files/dirs not existing